specification/sovereign/Sovereign.Management/main.tsp

```diff
@@ -0,0 +1,163 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-autorest";
+import "@azure-tools/typespec-azure-resource-manager";
+
+import "./common.tsp";
+import "./landingZoneAccountResourceProperties.tsp";
+import "./landingZoneConfigurationResourceProperties.tsp";
+import "./landingZoneRegistrationResourceProperties.tsp";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.ResourceManager;
+using Azure.Core;
+using OpenAPI;
+
+@service({
+  title: "Cloud for Sovereignty Resource Provider.",
+})
+@versioned(Versions)
+@armProviderNamespace
+@armCommonTypesVersion("v5")
+namespace Microsoft.Sovereign;
+
+interface Operations extends Azure.ResourceManager.Operations {}
+
+@doc("The Landing zone account parent resource type.")
```

Comment:
The Landing zone account resource type. A Landing zone account is the container for configuring, deploying and managing multiple landing zones.